Formula/oak.rb

```diff
@@ -0,0 +1,18 @@
+class Oak < Formula
+  desc "Expressive, simple, dynamic programming language"
+  homepage "https://oaklang.org/"
+  url "https://github.com/thesephist/oak/archive/v0.2.tar.gz"
+  sha256 "d06ba53f88490f5d8f70f45515afaf6ee8e4119c8ab477e3b4de8ef3cc7a2c05"
+  license "MIT"
+  head "https://github.com/thesephist/oak.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-s -w")
    (share/"vim/vimfiles/syntax").install "tools/oak.vim"
```

Reply:
Mulling it over, I'm not that eager to get the syntax file in the Homebrew distribution at the moment — it may be better distributed as its own Vim plugin or something. But I appreciate the suggestion! This is good to know in case I want it in the future.